neural_compressor/strategy/basic.py

```diff
@@ -179,38 +179,49 @@ def next_tune_cfg(self):
         from copy import deepcopy
         tuning_space = self.tuning_space
         calib_sampling_size_lst = tuning_space.root_item.get_option_by_name('calib_sampling_size').options
+        self.basic_last_quant_config = None
+        self.early_stopping = False
+        # flag to record whether need to the op-type-wise quantized have been done
+        self._optype_wise_quant_flag = self._quant_level == "auto"
```

Comment:
I didn't understand this logic. why it happens at basic.py? auto quant level shouldn't impact this part.